pyccel/codegen/printing/fcode.py

```diff
@@ -603,13 +602,15 @@ def _print_SumFunction(self, expr):
     def _print_PythonLen(self, expr):
         var = expr.arg
         idx = 1 if var.order == 'F' else var.rank
+        prec = self._print(expr.precision)
```

Comment:
This improvement is not implemented yet, you still need to use the `iso_c_binding` dictionary

Reply:
done